packages/core/contracts/cross-chain-oracle/GovernorHub.sol

```diff
@@ -49,9 +52,11 @@ contract GovernorHub is Ownable, Lockable {
         uint256 chainId,
         address to,
         bytes memory dataFromGovernor
-    ) external nonReentrant() onlyOwner {
+    ) external payable nonReentrant() onlyOwner {
         bytes memory dataSentToChild = abi.encode(to, dataFromGovernor);
-        messengers[chainId].sendMessageToChild(dataSentToChild);
+
+        // Pass all msg.value to Messenger
+        messengers[chainId].sendMessageToChild{ value: msg.value }(dataSentToChild);
```

Comment:
sheesh. ok this is painful. there is no compatibility issues with the `Governor.sol` sending transactions that contain a `value`, right? it can support this in its current form?

Reply:
Indeed the `Governor.sol`'s method includes associated `value` field to enable this: https://github.com/UMAprotocol/protocol/blob/a4e395f4a714f54f042078baac3993d339641ba6/packages/core/contracts/oracle/implementation/Governor.sol#L190

Reply:
is there a solution where rather than the `value` coming from the caller it is some value that is pre-loaded into the `GovernorHub` to pay for gas on L2? this way, we dont need to make this function payable and can rather let us drop some funds onto the bridge pool to pay for requests out of the contracts balance?

Reply:
the big advantage of doing it this way is that the `Governor` now no longer needs to pay for these actions. Moreover, the `Governor` does not need to consider the difference between the Arbitrum and other chain calls as the payment for the cross-chain actions is embedded within _this_ contract rather than the calling contract.

Reply:
Yeah good call out... I'm going to change the `Arbitrum_ParentMessenger` so that it can receive ETH and use its own balance to send to the inbox